scripts/icons.js

```diff
@@ -0,0 +1,18 @@
+const fs = require("fs");
+const path = require("path");
+
+const dir = path.resolve(__dirname, "../src/icons");
+const iconRegexp = new RegExp(/.*Icon\.tsx/);
+
+const files = fs.readdirSync(dir, {
+  withFileTypes: true,
+});
+
+console.log(files);
```

Comment:
*wild console log appears*

Reply:
I left it here on purpose, though I come to think it's redundant 🤔